.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -3,25 +3,25 @@ name: Bug report
 about: Create a report to help us improve
 ---
 
+**Platform (please complete the following information):**
+ - OS: [e.g. Windows 10]
+ - Version (e.g. Desktop Wallet 2.0.0)
+
 **Describe the bug**
 A clear and concise description of what the bug is.
 
-**To Reproduce**
-Steps to reproduce the behavior:
+**How to reproduce the bug**
+Steps to reproduce the behaviour:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
-**Expected behavior**
```

Comment:
I know it, but, since we decided to follow British English, I was trying to keep it consistent.
BTW, if you see typos of grammar mistakes on my issues, and if you don't mind, I'd love to hear your feedback.